modules/tensor_mechanics/src/materials/ComputeMultiPlasticityStress.C

```diff
@@ -17,7 +17,7 @@ InputParameters validParams<ComputeMultiPlasticityStress>()
   params += validParams<MultiPlasticityDebugger>();
   params.addClassDescription("Base class for multi-surface finite-strain plasticity");
   params.addRangeCheckedParam<unsigned int>("max_NR_iterations", 20, "max_NR_iterations>0", "Maximum number of Newton-Raphson iterations allowed");
-  params.addRequiredRangeCheckedParam<Real>("ep_plastic_tolerance", "ep_plastic_tolerance>0", "The Newton-Raphson process is only deemed converged if the plastic strain increment constraints have L2 norm less than this.");
+  params.addRequiredParam<Real>("ep_plastic_tolerance", "The Newton-Raphson process is only deemed converged if the plastic strain increment constraints have L2 norm less than this.");
```

Comment:
Any reason you removed the range check here? Negative tolerances fro L2 norm values don't really make sense.